lib/Profile.dart

```diff
@@ -0,0 +1,227 @@
+import 'dart:io';
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:image_picker/image_picker.dart';
+
+class Profile extends StatefulWidget {
+  Profile({Key key}) : super(key: key);
+  @override
+  _ProfileState createState() => _ProfileState();
+}
+
+class _ProfileState extends State<Profile> {
+  File _image;
+
+  Future getImage() async {
+    var image = await ImagePicker.pickImage(source: ImageSource.gallery);
+    setState(() {
+      _image = image;
+    });
```

Comment:
This should be fine for now, but keep in mind that the image (once the backend code is there) should be sent to the backend. I'll add "Profile Page Functionality" or something like that to the Monday.com board.

Reply:
I also recommend that you make "Changing the Profile Picture (Locally)" a different pull request. Could be nice to separate.